src/include/pbs_ifl.h

```diff
@@ -468,6 +468,7 @@ enum mgr_obj {
 #define PBS_MAXQUEUENAME	15		/* max queue name length */
 #define PBS_MAXJOBNAME  	230		/* max job name length */
 #define PBS_MAXSERVERNAME	PBS_MAXHOSTNAME	/* max server name length */
+#define MAX_SVR_ID (PBS_MAXSERVERNAME + 6)	/* max server name + port */
```

Comment:
Want to add the word "INSTANCE" to this variable?

Reply:
It's not used anymore so ill remove it